electricitymap/contrib/capacity_parsers/IRENA.py

```diff
@@ -0,0 +1,146 @@
+import json
+from datetime import datetime
+
+import pycountry
+from requests import Response, Session
+
+from electricitymap.contrib.config import ZoneKey
+
+"""The data is downloaded from the IRENA API. """
+
+IRENA_ZONES = ["IL", "IS", "LK", "NI", "GF", "PF"]
+
+IRENA_JSON_TO_MODE_MAPPING = {
+    0: "solar",
+    1: "solar",
+    2: "wind",
+    3: "wind",
+    4: "hydro",
+    5: "hydro",
+    6: "hydro storage",
+    7: "unknown",
+    8: "biomass",
+    9: "biomass",
+    10: "biomass",
+    11: "biomass",
+    12: "geothermal",
+    13: "coal",
+    14: "oil",
+    15: "gas",
+    16: "unknown",
+    17: "nuclear",
+    18: "unknown",
+}
+
+SPECIFIC_MODE_MAPPING = {"IS": {16: "oil"}}  # unknown -> oil
```

Comment:
Is it because IRENA returns this as unknown when we know it is oil? Maybe we could write a longer comment to explain that